cc-env_test.go

```diff
@@ -855,6 +923,72 @@ func TestCCEnvHandleSettings(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestCCEnvHandleSettingsInvalidShimConfig(t *testing.T) {
+	assert := assert.New(t)
+
+	tmpdir, err := ioutil.TempDir("", "")
+	if err != nil {
+		panic(err)
```

Comment:
`panic` feels a bit harsh - OK, I know this is like a moderately fatal system error, and not strictly a test failure, but I wonder if a `Fatal` from the test suite might be more appropriate? I defer to your knowledge on that though.

Reply:
Fixed - it was a cut'n'paste from other (older) tests that still do call `panic()` (see sprint idea above :)